data/locale/sv.ts

```diff
@@ -195,41 +195,41 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     </message>
     <message>
         <source>Loopback point:</source>
-        <translation>Loopback punkt:</translation>
+        <translation>Loopback-punkt:</translation>
     </message>
     <message>
         <source>With this knob you can set the point where the loop starts. </source>
-        <translation>Den här vreden ställer in vart loopen startar.</translation>
+        <translation>Med det här vredet så kan du ställa in var loopen börjar.</translation>
     </message>
 </context>
 <context>
     <name>AudioFileProcessorWaveView</name>
     <message>
         <source>Sample length:</source>
-        <translation>Ljudfilens längd:</translation>
+        <translation>Ljudklippets längd:</translation>
```

Comment:
"Ljudklippet" is a much better translation for sample than "Ljudfil", perhaps you should replace it everywhere?